static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -233,6 +234,13 @@ function useRefetchGroupForReprocessing({
   }, [hasReprocessingV2Feature, refetchGroup]);
 }
 
+function getUserDefault() {
+  const user = ConfigStore.get('user');
```

Comment:
Small thing, but we generally want to use `useLegacyStore` instead of ConfigStore directly. This is because `useLegacyStore` will react to config updates.

In order to do this you'll need to convert this to a hook: `useDefaultIssueEvent()`

And while you're at it, I think it would be good to extract this out into a util that can be used in both places.